platform/darwin/src/MGLGeoJSONSource.h

```diff
@@ -138,6 +152,10 @@ extern const MGLGeoJSONSourceOption MGLGeoJSONSourceOptionSimplificationToleranc
  */
 @property (nonatomic, nullable) NSURL *URL;
 
+/**
+ Returns all objects from the shape that conforms to `MGLFeature`.
+ */
+- (NSArray<MGLFeature> *)features;
```

Comment:
Use `NS_ARRAY_OF()` in public APIs for the benefit of developers compiling against the iOS 8 or OS X 10.10 SDKs.

Also, this method should be a property like before.

Finally, the return type should be an array of `id <MGLFeature>`, not an NSArray conforming to the MGLFeature protocol. There's a big difference.